allauth/account/forms.py

```diff
@@ -629,7 +625,8 @@ def _send_password_reset_mail(self, request, email, users, **kwargs):
 
             if app_settings.AUTHENTICATION_METHOD != AuthenticationMethod.EMAIL:
                 context["username"] = user_username(user)
-            get_adapter().send_mail("account/email/password_reset_key", email, context)
+            adapter.send_password_reset_mail(user, email, context)
```

Comment:
I tried to not pass `context` like this but it was pretty cumbersome to try and build that object in the adapter instead of here so I did it this way.

Reply:
@pennersr LMK if you'd like this reworked to avoid passing `context` around!

Reply:
This looks good to me :+1: